helix-query.yaml

```diff
@@ -51,3 +51,13 @@ indices:
         select: head > meta[property="article:tag"]
         values: |
           attribute(el, 'content')
+      image:
+        select: main .career-hero-photo img
+        value: attribute(el, "src")
+      quote:
+        select: main blockquote.career-hero-quote
+        value: textContent(el)
+      jobtitle:
+        select: main p.career-hero-title
+        value: textContent(el)
```

Comment:
should these be prefixed with `career-hero-` to indicate their `career` page specific meaning. For non-career pages, likely these would remain empty.

Reply:
Thanks! I prefixed it with just `career-` as while it was taken from the hero, it's used in another career component (the carousel) as input.